fhir-examples-generator/src/main/java/com/ibm/fhir/examples/CompleteMockDataCreator.java

```diff
@@ -186,8 +197,27 @@ public CompleteMockDataCreator() throws IOException {
                     /////////////////
                     // Special values
                     /////////////////
+                    // Must be a valid BCP-47 code (Code)
+                    if ((builder instanceof Attachment.Builder && "language".equals(method.getName()))
+                            || (builder instanceof CodeSystem.Concept.Designation.Builder && "language".equals(method.getName()))
+                            || (builder instanceof Resource.Builder && "language".equals(method.getName()))
+                            || (builder instanceof ValueSet.Compose.Include.Concept.Designation.Builder && "language".equals(method.getName()))) {
+                        argument = Code.of("en-US");
```

Comment:
pretty minor, but you could consider making "en-US" a constant since its used 3 times here...or potentially even make the `Code.of("en-US")` the constant and re-use it

Reply:
Updated it to a constant.